src/conv-core/cpuaffinity.C

```diff
@@ -415,8 +418,13 @@ typedef struct _affMsg {
 #endif
 } affMsg;
 
+static inline bool operator< (const skt_ip_t & a, const skt_ip_t & b)
+{
+  return memcmp(&a, &b, sizeof(a)) < 0;
+}
+
 static rankMsg *rankmsg = NULL;
-static CmmTable hostTable;
+static std::map<skt_ip_t, hostnameMsg *> hostTable;
```

Comment:
Could this also be an unorderd_map?

Reply:
Both could, but I would need to figure out how to implement the necessary hash function.

Reply:
Ok, might not be necessary then.